recipe/meta.yaml

```
@@ -30,11 +30,11 @@ requirements:
     - cython                                 # [build_platform != target_platform]
     - swig                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [(cuda_compiler or "None") != "None"]
```

nit: Would match quoting and group `compiler`s and `stdlib` together

```suggestion
    - {{ stdlib("c") }}
    - {{ compiler("c") }}
    - {{ compiler("cxx") }}
    - {{ compiler("cuda") }}  # [(cuda_compiler or "None") != "None"]
```